azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/model/AzureCredentialWrapper.java

```diff
@@ -16,8 +16,6 @@
 import java.util.List;
 import java.util.Objects;
 
-;
-
 public class AzureCredentialWrapper {
```

Comment:
```
        this.authMethod = method;
        this.tokenCredential = tokenCredential;
        this.env = env;
```
make these 3 field `final` and then you can use `@RequiredArgsConstructor` to simplify your code.